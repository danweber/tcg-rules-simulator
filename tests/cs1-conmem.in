VERSION:2
# Gain 9 memory. Then, draw 1 card.
FIELD1:Yellow,CS1-02
FIELD2:Suspend1,DW1-Van Suspend2,DW1-Van Safe,DW1-Van
GAME:P1 T3 MAIN 4
HAND1:CS1-Confusing CS1-Confusing
DECK1:CS1-02 CS1-02
DUMP:HAND

ECHO: -- Use option. End up at 4 - 12 + 9 = 1  and draw
CMD:PLAY CS1-Confusing
DUMP:HAND

ECHO: -- Try to use again. Can't, not enough memory.
CMD:PLAY CS1-Confusing
DUMP:HAND



