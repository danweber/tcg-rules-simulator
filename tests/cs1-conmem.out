FIELD1:Yellow,CS1-Papershield
FIELD2:Suspend1,DW1-Vanillamon Suspend2,DW1-Vanillamon Safe,DW1-Vanillamon
HAND1:CS1-Confusingmemory CS1-Confusingmemory
HAND2:
GAME:P1 T3 MAIN 4
ECHO: -- Use option. End up at 4 - 12 + 9 = 1  and draw
FIELD1:Yellow,CS1-Papershield
FIELD2:Suspend1,DW1-Vanillamon Suspend2,DW1-Vanillamon Safe,DW1-Vanillamon
HAND1:CS1-Papershield CS1-Confusingmemory
HAND2:
GAME:P1 T3 MAIN 1
ECHO: -- Try to use again. Can't, not enough memory.
FIELD1:Yellow,CS1-Papershield
FIELD2:Suspend1,DW1-Vanillamon Suspend2,DW1-Vanillamon Safe,DW1-Vanillamon
HAND1:CS1-Papershield CS1-Confusingmemory
HAND2:
GAME:P1 T3 MAIN 1
